hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/ProtobufLogReader.java

```diff
@@ -400,22 +400,22 @@ protected boolean readNext(Entry entry) throws IOException {
           initCause(realEofEx != null ? realEofEx : ex);
       }
       if (trailerPresent && this.inputStream.getPos() > this.walEditsStopOffset) {
-        LOG.error(
```

Comment:
This particular message should only happen if the read was expected to succeed because the trailer presence means the wal was closed. Is this specific message showing up in the false alarm?

Reply:
No, this one did not